tests/conftest.py

```
@@ -31,6 +31,14 @@
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
 
+@pytest.fixture(autouse=True)
+def reset_environment():
+    """Prevent environment variables from leaking into tests."""
+    for key in os.environ.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)